frontend/src/components/FileBrowser/FileBrowser.js

```diff
@@ -470,6 +469,7 @@ export class FileBrowserLite extends React.Component<
         this.state = {
             currentWorkingDirectory: '',
             fileBrowserData: {},
+            isVisible: !this.props.startCollapsed,
```

Comment:
This state doesn't seem to be in used anymore, let's remove it

Reply:
Removed the `isvisible` in FileBrowserLite